distribution/docs/src/main/jdocs/content/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,92 @@
+
+The rule is divided into two fields: "Criteria" and "Expected Value". The "Criteria" field features a drop-down list containing the four elements for which equality can be tested:
+
+* IP Address of where the ingest request came from
+* Host Name of where the ingest request came from
+* Scheme that the ingest request arrived on, for example, _http_ vs _https_
+* Context Path that the ingest request arrived on, for example, _/services/catalog_
+
+In order for a rule to evaluate to true and the attributes be applied, the value in the "Expected Value" field must be an exact match to the actual value of the selected criteria. For example, if the selected criteria is "IP Address" with an expected value of "192.168.0.1", the rule only evaluates to true for ingest requests coming from "192.168.0.1" and nowhere else.
+
+.Check for IPv6
+IMPORTANT: Verify your system's IP configuration. Rules using "IP Address" may need to be written in IPv6 format.
+
+The key-value pairs within each rule should take the following form: "key = value" where the "key" is the name of the attribute and the "value" is the value assigned to that attribute. Whitespace is ignored unless it is within the key or value. Multi-valued attributes can be expressed in comma-separated format if necessary.
+
+.Examples of Valid Attribute Assignments
+----
+contact.contributor-name = John Doe
+contact.contributor-email = john.doe@example.net
+language = English
+language = English, French, German
+security.access-groups = SJ202, SR 101, JS2201
+----
+
+include::{adoc-include}org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin-table-contents.adoc[]
+
+====== Useful Attributes
+
+The following table provides some useful attributes that may commonly be set by this plugin:
+
+.Useful Attributes
+|===
+|Attribute Name |Expected Format |Multi-Valued
+
+|expiration
+|ISO DateTime
+|no
+
+|description
+|Any String
+|no
+
+|metacard.owner
+|Any String
+|no
+
+|language
+|Any String
+|yes
+
+|security.access-groups
+|Any String
+|yes
+
+|security.access-individuals
+|Any String
+|yes
+|===
+
+===== Usage Limitations of the Metacard Ingest Network Plugin
+
+* This plugin only works for ingest (create requests) performed over a network; data ingested via command line does not get processed by this plugin.
+* Any attribute that is already set on the metacard will not be overwritten by the plugin.
+* The order of execution is not guaranteed. For any rule configuration where two or more rules add different values for the same attribute, it is undefined what the final value for that attribute will be in the case where more than one of those rules evaluates to true.
```

Comment:
I'll see what @Lambeaux has to add. Maybe we can just leave it at "it can be complicated to predict the final value for an attribute will be" ❓